_ml-commons-plugin/cluster-settings.md

```diff
@@ -62,7 +91,7 @@ plugins.ml_commons.sync_up_job_interval_in_seconds: 10
 
 ## Predict monitoring requests
 
-Controls how many upload model tasks can run in parallel on one node. If set to `0`, you cannot upload models to any node.
+Controls how many predict requests are monitored on one node. If set to `0`, OpenSearch clears all monitoring predict request in cache and does not monitor for new predict requests.
```

Comment:
"requests", right?
"If set to `0`, OpenSearch clears all monitoring predict requests in cache ..."

Reply:
Fixed